txp/Pos/Txp/Toil/Logic.hs

```diff
@@ -178,28 +176,20 @@ verifyGState curEpoch txAux txFee = do
 verifyBootEra
     :: forall ctx m .
        ( MonadError ToilVerFailure m
-       , HasLens' ctx GenesisStakeholders
+       , HasLens' ctx GenesisWStakeholders
        , MonadReader ctx m)
     => EpochIndex -> EpochIndex -> TxAux -> m ()
 verifyBootEra curEpoch unlockEpoch txAux = do
-    let bootEra = isBootstrapEra curEpoch unlockEpoch
-    bootHolders <- views (lensOf' @GenesisStakeholders) unGenesisStakeholders
+    let bootEra = isBootstrapEra unlockEpoch curEpoch
```

Comment:
Actually it would be better to propagate `MonadGState` here and use `gsIsBootstrapEra` here which takes only one argument, it would allow to avoid this bug. But this issue is not about it of course.